scripts/forms-scaffolder.js

```diff
@@ -0,0 +1,430 @@
+import {
```

Comment:
I'd prefer this script to be in tools folder since it is not rendered with forms block.